creator-node/test/lib/app.js

```diff
@@ -22,7 +24,9 @@ async function getApp (ipfsClient, libsClient, blacklistManager, ipfsLatestClien
     libs: libsClient,
     blacklistManager: blacklistManager,
     redis: redisClient,
-    monitoringQueue: new MonitoringQueueMock()
+    monitoringQueue: new MonitoringQueueMock(),
+    syncQueueService: new SyncQueueService(nodeConfig, redisClient, ipfsClient, ipfsLatestClient || ipfsClient),
```

Comment:
don't we need to pass in mockService registry object into syncQueueService here?

Reply:
good catch, fixed